bundle/next-all-namespaces/eclipse-che-preview-openshift/metadata/annotations.yaml

```diff
@@ -13,5 +13,4 @@ annotations:
   # Annotations for testing.
   operators.operatorframework.io.test.mediatype.v1: scorecard+v1
   operators.operatorframework.io.test.config.v1: tests/scorecard/
-
-  com.redhat.openshift.versions: "v4.8"
+\n  com.redhat.openshift.versions: "v4.8"
```

Comment:
I think it is autogenerated. (probably bug in our scripts)
But could you revert this?